src/BSeries.jl

```diff
@@ -612,6 +616,93 @@ function bseries(ros::RosenbrockMethod, order)
     return series
 end
 
+"""
+    ContinuousStageRungeKuttaMethod
+
+A struct that describes a CSRK method. This kind of 'struct' should be constructed 
+via [`CSRK`] 
+        'csrk = CSRK(M)' 
+in order to later call the 'bseries' function. 
+
+# References
+- Yuto Miyatake and John C. Butcher.
+"A characterization of energy-preserving methods and the construction of
+parallel integrators for Hamiltonian systems."
+SIAM Journal on Numerical Analysis 54, no. 3 (2016): 
+[DOI: 10.1137/15M1020861](https://doi.org/10.1137/15M1020861) 
+"""
+struct ContinuousStageRungeKuttaMethod{MatT <: AbstractMatrix}
+    matrix::MatT
+end
+
+
+"""
+    bseries(csrk::ContinuousStageRungeKuttaMethod, order)
+
+Compute the B-series of the [`ContinuousStageRungeKuttaMethod`](@ref) `csrk`
+up to the prescribed integer `order` as described by Miyatake & Butcher (2015).
+    
+!!! note "Normalization by elementary differentials"
+The coefficients of the B-series returned by this method need to be
+multiplied by a power of the time step divided by the `symmetry` of the
+rooted tree and multiplied by the corresponding elementary differential
+of the input vector field ``f``.
+See also [`evaluate`](@ref).
+
+# Example:
+The energy-preserving 4x4 matrix given by Miyatake & Butcher (2015) is
+```
+M = [-6//5   72//5  -36//1  24//1;
+        72//5  -144//5 -48//1  72//1;
+        -36//1 -48//1   720//1 -720//1;
+        24//1   72//1  -720//1  720//1]
+```
+
+Then, we calculate the bseries with the following code:
+
+```
+csrk = CSRK(M)
+series = bseries(csrk, 4)
+TruncatedBSeries{RootedTree{Int64, Vector{Int64}}, Rational{Int64}} with 9 entries:
+  RootedTree{Int64}: Int64[]      => 1//1
+  RootedTree{Int64}: [1]          => 1//1
+  RootedTree{Int64}: [1, 2]       => 1//2
+  RootedTree{Int64}: [1, 2, 3]    => 6004799503160661//36028797018963968
+  RootedTree{Int64}: [1, 2, 2]    => 6004799503160661//18014398509481984
+  RootedTree{Int64}: [1, 2, 3, 4] => 6004799503160661//144115188075855872
+  RootedTree{Int64}: [1, 2, 3, 3] => 6004799503160661//72057594037927936
```

Comment:
No, sorry: seems like there is something happening with the definition of `V` in 
```
series = TruncatedBSeries{RootedTree{Int, Vector{Int}}, V}()
series[rootedtree(Int[])] = one(V)
``` 
so that, although the output of `elementary_differentials_csrk(csrk, t)` is correct, for some values it will throw an approximation to the original result instead of saving the value of `elementary_differentials_csrk(csrk, t)` Do you know how can I fix it?

Reply:
Seems like the numerical function `N()` solves this issue, but I don't know if it has limitations for future applications of this code (since this is the first time I'm working with this function).

Reply:
Let's see. First, please add more tests etc. Then, possible problem can show up already